### ISSUE TITLE
Running on Intranet

### ISSUE THREAD

**Issue by Fastidious** [firehol/netdata#1752]
Where I work, most of our servers do not have access to the outside world. What potential issues could I foresee? I know update checking will not work, nor alarms that use third party services, nor the global registry. Is there anything else?

Thanks!

**Comment by ktsaou** [comment]
there are a lot of people using netdata like that.

## netdata servers

Only notifications using external services will fail, but if you use an outbound http/https proxy, you can set it for them and they will work.

## your web browser

If your web browser has internet access, everything else will work.

If not even your web browser has access to the internet:

1. You can make one of your netdata your local registry so that `my-netdata` menu will work. You will have to set its URL on all your netdata installations.

2. Update checking will not work.

**Comment by Fastidious** [comment]
> You can make one of your netdata your local registry so that my-netdata menu will work. You will have to set its URL on all your netdata installations.

Could you provide a link on how to accomplish that? Thank you much.

**Comment by ktsaou** [comment]
Of course: https://github.com/firehol/netdata/wiki/mynetdata-menu-item#every-netdata-can-be-a-registry

**Comment by ktsaou** [comment]
ok. I am closing this. If you need help, just post.

**Comment by jneivil** [comment]
I have 2 servers (server1 and server2)

I installed in both netdata and all its ok.

But both servers is in my intranet. I can't use registry.

In server1 (master) I've modified /etc/netdata/netdata.conf
[registry]
registry to announce = http://server1:19999

In server2 
[registry]
registry to announce = http://server1:19999

I reboot both servers but in http://server1:19999 or http://server2:19999 I cant see another server. Only see london, new york .... (in my intranet those servers don't have sense)

Neither I can't see in both servers: web url: my-netdata -> what is this -> YOUR_PERSONAL_ID

But in /var/lib/netdata/registry/netdata.public.unique.id exists and it has content 

How I can remove london, new york , ....?
How I can use server1 and serve2 registry ?
Why I dont see my personal id?

Thanks
